doc/tutorials/space_shooter/step_6.md

```diff
@@ -29,19 +29,46 @@ the `RectangleHitbox`, which like the name implies will make a rectangular area
 hitbox.
 
 Hitboxes are also components, so in order to add them to our components we can simply add
-the following line on both the `Bullet`'s and `Enemy`'s `onLoad` method:
+them to the components that we want to have hitboxes, so let's do it, let's start by adding the
+following line to the `Enemy` class:
 
 ```dart
 add(RectangleHitbox());
 ```
 
+For the bullet we will do the same, but with a slight difference:
+
+```dart
+add(
+  RectangleHitbox(
+    collisionType: CollisionType.passive,
+  ),
+);
+```
+
+The `collisionType`s are very important to understand, since they can directly impact the game
+perfomance!
+
+There are three types of collisions in Flame:
+- `active`: These hitboxes will both check itself against other hitboxes, as well other hitboxes 
```

Comment:
Maybe the list here can just be copied?
https://docs.flame-engine.org/latest/flame/collision_detection.html#collisiontype

It has slightly better wording.